app/models/jsonrates.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonrates
+  attr_accessor :action, :currencies
```

Comment:
[![Documentation](https://www.pullreview.com/assets/documentation-small.png)](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3) Add documentation to the following class[ `Jsonrates`](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3)